project/plugins.sbt

```diff
@@ -1,15 +1,10 @@
 // The Typesafe repository
 resolvers += "Typesafe repository" at "https://repo.typesafe.com/typesafe/releases/"
-resolvers += "JBoss" at "https://repository.jboss.org" // needed for the scrooge-sbt-plugin due to broken jboss deps
 
-libraryDependencies += "org.vafer" % "jdeb" % "1.6" artifacts (Artifact("jdeb", "jar", "jar"))
+libraryDependencies += "org.vafer" % "jdeb" % "1.10" artifacts (Artifact("jdeb", "jar", "jar"))
 
 addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.8.2")
 
-addSbtPlugin("io.get-coursier" % "sbt-coursier" % "1.1.0-M7")
+addSbtPlugin("com.gu" % "sbt-riffraff-artifact" % "1.1.18")
 
-addSbtPlugin("com.twitter" % "scrooge-sbt-plugin" % "18.4.0")
```

Comment:
is this just leftover from a time when we thought giant might want to eat some thrift?

Reply:
Yeah Giant did once have Thrift :)

Simon wrote some performance tracking on the ingestion pipeline that was stored as Thrift files in S3.

My guess is that we forgot to remove this when we removed the performance stuff.